src/EnergyPlus/Data/EnergyPlusData.cc

```diff
@@ -312,6 +312,7 @@ EnergyPlusData::EnergyPlusData()
 
 void EnergyPlusData::clear_state()
 {
+    this->ready = true;
```

Comment:
A new flag is added to make sure someone doesn't try to re-run an EnergyPlus simulation with an already used state.  The same state can be used to make API calls routinely, but you can't kick off a second simulation run without resetting the state.